lib/flatpak-game.lua

```diff
@@ -65,6 +65,7 @@ function identify(file)
 end
 
 function verify_gog(file)
+    print('Verifying game file.')
```

Comment:
Which indentation convention are you following? I see tab characters, but also some indentations that are spaces, and a few that are tabs *and* spaces, while the Lua documentation says to use two spaces.